### ISSUE TITLE
Always got empty items array for GET path array response with unique items

### ISSUE THREAD

**Issue by gschafra** [swagger-mock/swagger-mock#13]
Since merge of PR #12, I've always got an empty array returned from my GET endpoint path with unique items configured:

```yaml
"/idds/{id}/invoices/{doctorsInvoiceNumber}/documents":
    get:
      summary: List of documents for an IDD specific invoice
      ...
    responses:
        "200":
          description: List of documents
          content:
            application/json:
              schema:
                type: array
                uniqueItems: true
                items:
                  $ref: "#/components/schemas/File"
```
Find full spec attached

**Comment by strider2038** [comment]
@gschafra thank you. Can you give me a full specification? Or at least definition of `#/components/schemas/File`? I cannot reproduce an issue.

**Comment by gschafra** [comment]
@strider2038: Find the specification here: [REST_API_OAS3.zip](https://github.com/swagger-mock/swagger-mock/files/3414885/REST_API_OAS3.zip). Maybe I just missed something. Many thanks in advanced!

**Comment by strider2038** [comment]
@gschafra the problem is that File has `type: object` and it is detected as object without properties, not as combined type `allOf`. I will fix default behaviour of parser to detect combined types first. 

```yaml
File:
      description: File object
      type: object # <---
      allOf:
```

**Comment by strider2038** [comment]
Issue fixed in [v0.1.7](https://github.com/swagger-mock/swagger-mock/releases/tag/v0.1.7)
